backend/handlers/setup/setuphandler_test.go

```diff
@@ -0,0 +1,90 @@
+package setup
+
+import (
+	"encoding/json"
+	"net/http"
+	"net/http/httptest"
+	"socialpredict/setup"
+	"strings"
+	"testing"
+)
+
+var loadEconomicsConfig = setup.LoadEconomicsConfig
+
+func TestGetSetupHandler(t *testing.T) {
+	tests := []struct {
+		Name             string
+		MockConfigLoader func() (*setup.EconomicConfig, error)
+		ExpectedStatus   int
+		ExpectedResponse string
+		IsJSONResponse   bool
+	}{
+		{
+			Name: "successful load",
+			MockConfigLoader: func() (*setup.EconomicConfig, error) {
+				return setup.LoadEconomicsConfig()
+			},
```

Comment:
I like the way you did this! Can we add an error case as well, where the config fails to load?

Reply:
Added and tagged a ticket within refactoring for this.

https://github.com/orgs/openpredictionmarkets/projects/6/views/1?pane=issue&itemId=76289411